numpy/core/_add_newdocs.py

```diff
@@ -1302,9 +1302,6 @@
     For integer arguments the function is equivalent to the Python built-in
     `range` function, but returns an ndarray rather than a list.
 
-    When using a non-integer step, such as 0.1, the results will often not
-    be consistent.  It is better to use `numpy.linspace` for these cases.
-
```

Comment:
I would be in favor of leaving this in, perhaps replacing the last sentence with "see Warnings for more details".

Reply:
reverted/updated